crates/proof-of-sql/src/sql/proof_plans/mod.rs

```diff
@@ -36,11 +36,15 @@ pub(crate) use group_by_exec::GroupByExec;
 mod group_by_exec_test;
 
 mod slice_exec;
-#[allow(unused_imports)]
 pub(crate) use slice_exec::SliceExec;
 #[cfg(all(test, feature = "blitzar"))]
 mod slice_exec_test;
 
+mod union_exec;
+pub(crate) use union_exec::UnionExec;
+#[cfg(all(test, feature = "blitzar"))]
```

Comment:
Let's not require `blitzar`.

Reply:
This isn't possible until we can replace blitzar here https://github.com/spaceandtimelabs/sxt-proof-of-sql/blob/main/crates/proof-of-sql/src/sql/proof/verifiable_query_result_test_utility.rs#L33 or we won't be able to exercise verification.